### PR TITLE
fix meshing in conda environments and renable tests

### PR DESCRIPTION
**Pull request by cekees**
# Mandatory Checklist

Please ensure that the following criteria are met:

- [x] Title of pull request describes the changes/features
- [x] Request at least 2 reviewers
- [x] If new files are being added, the files are no larger than 100kB. Post the file sizes.
- [x] Code coverage did not decrease. If this is a bug fix, a test should cover that bug fix. If a new feature is added, a test should be made to cover that feature.
- [x] New features have appropriate documentation strings (readable by sphinx)
- [x] Contributor has read and agreed with CONTRIBUTING.md and has added themselves to CONTRIBUTORS.md 

As a general rule of thumb, try to follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines.

# Description

### CODE REVIEW

File: proteus/TwoPhaseFlow/utils/Parameters.py [hunk 1]
```diff
@@ -1762,14 +1762,9 @@ def _initializeNumerics(self):
...
         prefix = self.n.linear_solver_options_prefix
-        if self._Problem.useSuperlu:
-            self.OptDB.setValue(prefix+'ksp_type', 'preonly')
-            self.OptDB.setValue(prefix+'pc_type', 'lu')
-            self.OptDB.setValue(prefix+'pc_factor_mat_solver_type', 'superlu_dist')
-        else:
-            self.OptDB.setValue(prefix+'ksp_type', 'cg')
-            self.OptDB.setValue(prefix+'pc_type', 'gamg')
-            self.OptDB.setValue(prefix+'ksp_max_it', 2000)
+        self.OptDB.setValue(prefix+'ksp_type', 'cg')
```
Comment (ejtovar) [inline proteus/TwoPhaseFlow/utils/Parameters.py:1765]: I'm not really familiar with the solver options for the TwoPhaseFlow models, so can you explain what is going on here @cekees? Why were the `useSuperlu` options removed?

**Review by ejtovar — COMMENTED** [review]
Looks like the `proteus/tests/proteus_tutorial/tests/test_floating_body.py` is still failing on the OSX build with `nan`'s in the Added Mass matrix. Is this a solver issue?

File: proteus/TwoPhaseFlow/utils/Parameters.py (hunk 1 above)
Comment (cekees) [reply proteus/TwoPhaseFlow/utils/Parameters.py:1765]: The added mass model here is a pure Neumann problem for the Poisson equation so it has the constant vectors in the null space.  I think generally it's safer to use an iterative solver that knows about the null space to solve this type of singlar problem.  It seems that superlu can handle it sometimes, but I'm not familiar with the details on how it's doing it. Unfortunately, this does not seem to be the issue with the travis osx test failure in any case.

Comment (cekees) [reply proteus/TwoPhaseFlow/utils/Parameters.py:1768]: Turns out it was the issue... The way the TwoPhaseFlow app is implemented right now you have to override this LU option in the child class _initializeNumerics funciton. /cc @zhang-alvin

**Comment by ejtovar** [comment]
Seems like everything is good to go now that the tests are passing!

**Review by ejtovar — APPROVED** [review]

**Pull request merged**